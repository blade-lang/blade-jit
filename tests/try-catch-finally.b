try {
  var a = 20
  raise new Error('Something happened')
} catch e {
  echo e.stacktrace
} finally {
  echo 'Cleaning up'
}