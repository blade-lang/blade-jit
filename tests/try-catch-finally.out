>>>
Error: Something happened\n\s+at.*try\-catch\-finally\.b:3:3 \-\> @\.script\(\)\nCleaning up